normandy/recipes/api/serializers.py

```diff
@@ -37,6 +38,25 @@ class Meta:
         ]
 
 
-class BundleSerializer(serializers.Serializer):
+class BundleSerializer(serializers.ModelSerializer):
     recipes = RecipeSerializer(many=True)
     country = serializers.CharField()
+
+
+class RecipeVersionSerializer(serializers.ModelSerializer):
+    date_created = serializers.SerializerMethodField(read_only=True)
+    recipe = serializers.SerializerMethodField(read_only=True)
```

Comment:
And then you can leave out the `get_date_created` and `get_recipe` methods below. Serializers are pretty smart about how to thread all the context and data objects around.